docs/openapi-docs/src/test/scala/sttp/tapir/docs/openapi/dtos/Items.scala

```diff
@@ -0,0 +1,10 @@
+package sttp.tapir.docs.openapi.dtos
+
+import sttp.tapir.{Schema, Validator}
+
+final case class Items[A](items: List[A], total: Int)
+
+object Items {
+  implicit def schema[A: Schema]: Schema[Items[A]] = Schema.derived[Items[A]]
+  implicit def validator[A: Schema]: Validator[Items[A]] = schema.validator
```

Comment:
the validator is not needed. I'd also move the schema implicit to the test, like in other tests